diagnostic.md

```diff
@@ -4,7 +4,10 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+.branch-infobar
+.text
+li
+h1
```

Comment:
I'm not sure where you're looking, but we are looking for full selector names. A few examples are `.text-gray`, or `header header-logged-in true`